float32 battery_voltage
uint16 cpu_load
bool compass_enabled
bool chksum_error
bool flying
bool motors_on
int16 flight_mode
uint16 up_time